example/index.html

```diff
@@ -58,11 +58,13 @@
   </style>
 
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
+  <script src="index.js" type="text/javascript"></script>
```

Comment:
The Javascript should not be split up, so i'd say either have it all in the `index.html` file or in the new `index.js` file. My preference is on the latter :).